cpp/farm_ng/core/prototools/event_log_reader.cpp

```diff
@@ -25,130 +25,95 @@
 
 namespace farm_ng {
 
-class EventLogReaderBinaryImpl : public EventLogReaderImpl {
+class EventLogReaderImpl : public EventLogReaderBase {
  public:
-  EventLogReaderBinaryImpl(std::filesystem::path const& log_path)
-      : log_path(log_path) {
-    reset();
+  EventLogReaderImpl() : instream_(), log_path_(), index_() {}
```

Comment:
Does any client code use this or can it be made private? 
The API is a bit confusing for this code path otherwise.

Reply:
okay, wrapping EventLogReaderImpl in an annom namespace...